pkg/api/hvpa_types.go

```diff
@@ -0,0 +1,23 @@
+package api
```

Comment:
license header (I don't repeat this for other files, so please check)

Reply:
Licence header added to all files.